examples/pubsub-simple/README.md

```diff
@@ -26,11 +27,14 @@ Run the following command in a terminal/command prompt:
 <!-- STEP
 name: Run subscriber
 expected_stdout_lines:
+  - '== APP == Subscriber received: id=1, message="hello world", content_type="application/json"'
+  - 'RETRY status returned from app while processing pub/sub event'
   - '== APP == Subscriber received: id=1, message="hello world", content_type="application/json"'
```

Comment:
This test is currently failing - therefore we see that this RETRY isn't actually working. ID=1 is not sent a second item.

Reply:
That's... interesting. I'm currently running my tests in a github codespaces instance, and tests pass there. Do you have any suggestions on how to better exercise this? Thanks.

Reply:
How are you running this particular test? Also what version of `daprd` are you running this against?